src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlNotificationTest/SqlNotificationTest.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Data.SqlClient.ManualTesting.Tests
     public class SqlNotificationTest : IDisposable
     {
         // Misc constants
-        private const int CALLBACK_TIMEOUT = 5000; // milliseconds
+        private const int CALLBACK_TIMEOUT = 6000; // milliseconds
```

Comment:
Any special reason for this change?

Reply:
these tests were failing on my local machine due to this time out and changing it made them pass. Increasing the wait time made them pass. I think it is up to resources available to the machine that tests are running at. I have reverted the change.

Reply:
reverted.